caproto/server/common.py

```diff
@@ -446,15 +449,15 @@ def __getitem__(self, pvname):
         addresses = get_beacon_address_list()
 
         while True:
-            if self.udp_sock is None:
+            if self.beacon_sock is None:
                 await self.async_layer.library.sleep(.1)
                 continue
             beacon = ca.RsrvIsUpResponse(13, self.port, self.beacon_count,
-                                         self.host)
+                                         self.interfaces[0])  # TODO which one?
```

Comment:
Important TODO here - I think it should probably indicate the IP for each interface in the beacon message.

Reply:
Does that mean we want a separate beacon loop (and `beacon_sock`) for each interface?